src/worker/WorkerSubscriptionPool.js

```diff
@@ -0,0 +1,35 @@
+class WorkerSubscriptionPool {
+  workers = new Map()
+  addWorker = ({ app, connection, worker }) => {
```

Comment:
Extreme nitpicking :haircut_man: since we only instanciate this class once, but I was wondering if there was a reason why the methods are not defined on the prototype?

Reply:
No particular reason... I may have been confused before with react to use arrow functions to bind `this`. Fixed in d70450d.